### PR TITLE
Base skeleton-module

### PR DESCRIPTION
**Pull request by concept-core**
This repo holds the main code for a base package.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+[*]
+charset = utf-8
+indent_size = 4
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (jordyvandomselaar) [inline .editorconfig]: Please change this to `true`, that way most editors will automatically remove whitespace at the end of lines.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+build
+composer.lock
```
Comment (jordyvandomselaar) [inline .gitignore]: Lock file should be committed to prevent different package versions from being installed across devices.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+docs
```
Comment (jordyvandomselaar) [inline .gitignore]: Why wouldn't we want to push documentation? And more importantly; why would we have this folder in the first place :O

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to `:package_name` will be documented in this file
```
Comment (jordyvandomselaar) [inline CHANGELOG.md]: Please change `:package_name` :P

Comment (concept-core) [reply CHANGELOG.md]: This is a whole repo is to scaffold for a new package so this is correct and should not be changed.

File: .gitignore (hunk 2 above)
Comment (concept-core) [reply .gitignore]: For the skeleton module it might smart to keep the lock file in here.

**Pull request merged**